src/listpack.c

```diff
@@ -1071,8 +1105,7 @@ void lpRandomPair(unsigned char *lp, unsigned long total_count, listpackEntry *k
 
     /* Generate even numbers, because listpack saved K-V pair */
     int r = (rand() % total_count) * 2;
-    p = lpSeek(lp, r);
-    assert(p);
+    assert((p = lpSeek(lp, r)));
     key->sval = lpGet(p, &vlen, NULL);
     if (key->sval) {
         key->slen = vlen;
```

Comment:
these can also be replaced with the new lpGetValue, as well as the ones in lpRandomPairsUnique.
or

Reply:
Fixed